docs/docusaurus/i18n/en.json

```diff
@@ -63,14 +63,14 @@
       "docs/docs_overview": {
         "title": "Documentation Overview"
       },
-      "dp/architecture_overview": {
-        "title": "Overview"
+      "dp/architecture_overview title: Overview hide_title: true": {
+        "title": "dp/architecture_overview title: Overview hide_title: true"
       },
-      "dp/debug_logs": {
-        "title": "View Logs"
+      "dp/debug_logs title: View Logs hide_title: true": {
+        "title": "dp/debug_logs title: View Logs hide_title: true"
       },
-      "dp/deploy_build": {
-        "title": "Build DP"
+      "dp/deploy_build title: Build DP hide_title: true": {
+        "title": "dp/deploy_build title: Build DP hide_title: true"
```

Comment:
why those changes on DP?

Reply:
Till "make precommit" it was fine showing only 2 line change.
Other changes started showing when executed "make dev". Should I remove it ?

Reply:
Oh that code is autogenerated. So probably it was not committed or the generator has been updated.

It is fine, you can leave it as is